dvc/utils/__init__.py

```diff
@@ -252,11 +252,19 @@ def load_stage_file_fobj(fobj, path):
     from dvc.exceptions import StageFileCorruptedError
 
     try:
-        return yaml.safe_load(fobj) or {}
-    except ScannerError:
+        yaml = YAML()
+        return yaml.load(fobj) or {}
+    except YAMLError:
         raise StageFileCorruptedError(path)
 
 
+def dump_stage_file(path, data):
```

Comment:
Should we move functions related to stage from here to `dvc/stage.py`?

Reply:
@Suor ?

Reply:
`dvc/stage.py` is big enough and making a new file just for that is questionable.

Reply:
how about `dvc/stage/utils.py`? It just seems wrong to have stage-specific logic in common utils.

Reply:
This will make `dvc.stage` into package, move `stage.py` to `__init__.py`, which I don't really like. And anyway this looks like refactoring, which doesn't belong here.

Reply:
Introduce `utils/stage.py` for now and create an issue to refactor utils.py? That file is probably bigger than `stage.py` now. Let's not silently increase tech debt at least.

Reply:
`dvc/stage.py` is second biggest now after `test_repro.py`, so no). I will move those to `utils/stage.py`.